### PR TITLE
chore(STONEINTG-937): add nightly job to clean GitLab webhooks

### PR DESCRIPTION
**Pull request by dheerajodha**

### CODE REVIEW

**Comment by dheerajodha** [comment]
/pj-rehearse periodic-ci-konflux-ci-e2e-tests-main-appstudio-dump-external-resources

**Comment by dheerajodha** [comment]
/pj-rehearse periodic-ci-konflux-ci-e2e-tests-main-appstudio-dump-external-resources

**Comment by dheerajodha** [comment]
/pj-rehearse periodic-ci-konflux-ci-e2e-tests-main-appstudio-dump-external-resources

**Review by dirgim — APPROVED** [review]
/lgtm if it helps :/

**Review by kasemAlem — APPROVED** [review]

**Comment by psturc** [comment]
/pj-rehearse periodic-ci-konflux-ci-e2e-tests-main-appstudio-dump-external-resources

**Comment by dheerajodha** [comment]
/pj-rehearse periodic-ci-konflux-ci-e2e-tests-main-appstudio-dump-external-resources

File: ci-operator/step-registry/redhat-appstudio/clean-external-resources/redhat-appstudio-clean-external-resources-commands.sh [hunk 1]
```diff
@@ -43,16 +47,17 @@ echo -e "[INFO] Start cleanup with user: ${GITHUB_USER}"
 
 cd "$(mktemp -d)"
 
-git clone --origin upstream --branch main "https://${GITHUB_TOKEN}@github.com/konflux-ci/e2e-tests.git" .
+git clone --origin testing --branch STONEINTG-937-nightly-job "https://${GITHUB_TOKEN}@github.com/kasemAlem/e2e-tests.git" .
```
Comment (psturc) [inline ci-operator/step-registry/redhat-appstudio/clean-external-resources/redhat-appstudio-clean-external-resources-commands.sh:50]: this needs to be reverted before merge

Comment (dheerajodha) [reply ci-operator/step-registry/redhat-appstudio/clean-external-resources/redhat-appstudio-clean-external-resources-commands.sh]: Reverted

**Pull request merged**